src/views/accounts/Automation/BaseExecutionList.vue

```diff
@@ -121,6 +121,20 @@ export default {
                     vm.visible = true
                     vm.reportUrl = `${this.url}${row.id}/report/`
                   }
+                },
+                {
+                  name: 'record',
+                  title: this.$t('Record'),
+                  can: this.$hasPerm('accounts.view_changesecretrecord'),
+                  callback: function({ row }) {
+                    return this.$router.push({
+                      name: 'AccountChangeSecretList',
+                      query: {
+                        tab: 'ChangeSecretRecord',
+                        execution_id: row.id
+                      }
+                    })
+                  }
                 }
               ]
             }
```

Comment:
There appear to be some syntax errors (like the extra closing brace) which should also be fixed. Otherwise everything looks good with no clear irregularities or potential issues. No further optimizations suggested based on the provided code snippet here.

If you have specific details about what you want these changes made for, please feel free to ask!